lib/mwUtil.js

```diff
@@ -33,6 +33,15 @@ mwUtil.makeETag = (rev, tid, suffix) => {
  * Normalizes the request.params.title and returns it back
  */
 mwUtil.normalizeTitle = (hyper, req, title) =>  mwUtil.getSiteInfo(hyper, req)
+.catch((e) => {
```

Comment:
My concern was that not all uses of that module might use a domain that is expected to exist. If the domain is user-controlled (not configured in RB), then the 500 might not be helpful. Should we just make & document that assumption?